### DISCUSSION TITLE
Previous Copilot Free Trial prevents successful activation of Github Global Campus student benefits

### DISCUSSION THREAD

**Discussion by Nomercyme** [community/community#110481] [Copilot Conversations]
### Select Topic Area

Bug

### Body

In the past I've used the free trial for Github Copilot on my personal account here. The free trial version has finished a while ago, however I'm still a student. 

I successfully applied to become a GitHub Global Campus student by adding my student email to this personal account and now wish to redeem the free Copilot service but am unable to do so. I get the following error message in Visual Studio Code: 
`Thank you for using GitHub Copilot. Your subscription has ended. You are currently logged in as [USERNAME]`

I haven't see any fix to this on the discussion forums. 

Thanks for your help!

**Comment by ZuhairTarif** [comment]
Apply for the GitHub Student Pack with your student email and if you are using vscode then download the copilot extension and login with GitHub. Hope it will resolve your problem. Have a nice day.
